### PR TITLE
[JENKINS-70809] - Fix "delete build" button overflow the side-panel

### PR DESCRIPTION
**Pull request by mamh2021**
See [JENKINS-70809](https://issues.jenkins.io/browse/JENKINS-70809)

### Proposed changelog entries

- Fix "delete build" button text overflow.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7735"></a>

### CODE REVIEW

**Review by NotMyFault — COMMENTED** [review]
Hey,

Thanks for proposing a fix for the issue risen. What do you think about my changes submitted? Instead of simply restoring the original behavior, we could display the full name of the build to delete, instead of abbreviating it after a few characters.

File: war/src/main/less/modules/side-panel-tasks.less
Comment (mamh2021) [reply war/src/main/less/modules/side-panel-tasks.less]: I agree with you very much, I also wanted to modify it like this before.thank you.

**Comment by mamh2021** [comment]
Another question, I'm not sure.
in file `war/src/main/less/modules/side-panel-tasks.less`,  there are two same "`.task-link-text` " at line ` 109` and `113`.
should one be deleted?


```
   109	  .task-link-text {
   110	    display: contents;
   111	  }
   112	
   113	  .task-link-text {
   114	    display: contents;
   115	    word-break: break-word;
   116	  }
```

**Comment by NotMyFault** [comment]
> there are two same ".task-link-text

Previously, we had 
```
  .task-link-text {
    display: contents;
  }

  .task-link-text {
    display: contents;
  }
```
which is declared twice. I removed the top one via browsertools and didn't notice anything odd, so I'd say we can safely remove it, unless there's some context I am missing. @janfaracik any idea?

**Review by NotMyFault — APPROVED** [review]
> there are two same ".task-link-text

Previously, we had 
```
  .task-link-text {
    display: contents;
  }

  .task-link-text {
    display: contents;
  }
```
which is declared twice. I removed the top one via browsertools and didn't notice anything odd, so I'd say we can safely remove it, unless there's some context I am missing. @janfaracik any idea?

**Review by janfaracik — APPROVED** [review]

**Comment by NotMyFault** [comment]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
